bindings/bind_py_structure_manager.cc

```diff
@@ -458,6 +458,132 @@ namespace rascal {
         py::call_guard<py::gil_scoped_release>());
   }
 
+  /**
+   * Bind a getter for the feature matrix comming from BlockSparseProperty.
+   *
+   * it will return a dictionary with keys->atomic number vals->feature mat
+   * the feature matrices size is (n_centers, inner_size)
```

Comment:
in a dictionary you have keys and values and the -> associates them with what they correspond to.

Reply:
Oh, OK I understand what you mean.

I would write this as `atomic number => features` or `associating atomic number to features` instead, I've usually seen the arrow used to symbolize the association itself. So I was reading this as a dict from keys to atomic number and another one from vals to features.

Reply:
I would suggest a small edit to `it will return a dictionary with items corresponding to atomic numbers and the associated feature matrix" for clarity

Reply:
please tell me if you are satisfied with the current edit